src/components/home/example.tsx

```diff
@@ -1,9 +1,19 @@
 const example = () => {
   return (
-    <div className="flex items-center justify-center text-[10vw] text-cvdsa-red-200">
-      example
+    <div className="
+    text-5x1/48.76
+    font-cvdsa-Montserrat-sans
+    font-[500]
+    flex h-screen w-screen items-center text-center mx-auto justify-center 
+    text-[4vw] 
+    text-cvdsa-white-255">
```

Comment:
```suggestion
    <div className="
    flex h-screen w-screen items-center text-center mx-auto justify-center 
    text-[4vw] ">
```
just a rough approximation of the font is fine, it doesn't need to be crazy. if text-5xl looks close enough to the figma, you can use that. montserrat is the default font, remove those styles. the color white is already added within tailwind as text-white